frontends/web/src/locales/en/app.json

```diff
@@ -240,7 +240,7 @@
       "nameLabel": "BitBox02 name",
       "namePlaceholder": "My BitBox02",
       "title": "Choose BitBox02 name",
-      "toastMicroSD": "Please make sure your microSD card is inserted in your BitBox02."
+      "toastMicroSD": "Please insert your microSD card into your BitBox02 which will be used to create a backup of the wallet."
```

Comment:
not changed in locize yet. will manually update the translation when this is reviewed and merged.

Reply:
makes sense, changed.